app/controllers/gardens_controller.rb

```diff
@@ -45,8 +45,7 @@ def edit
   # POST /gardens
   # POST /gardens.json
   def create
-    params[:garden][:owner_id] = current_member.id
-    @garden = Garden.new(garden_params)
+    @garden.owner_id = current_member.id
```

Comment:
ALSO -- we don't need to call `Model.new` in our controllers. the `load_and_authorize_resource` already does this on `#new`